### DISCUSSION TITLE
Hello~ I made some shapes :)

### DISCUSSION THREAD

**Discussion by SMK-SonHR** [exult/exult#407] [General]
![울티마 인장](https://github.com/exult/exult/assets/152712037/4022258f-6179-4bbe-bfde-c4bc7ffeab27)

Wow, New Area,,, (I`m korean, and I'm not good at English, Sorry )

I made some shapes and I wanna share these things,  ^^

Here are. 

![이미지_090](https://github.com/exult/exult/assets/152712037/9711fdcf-c7a9-4066-adb6-5617f968ecc9)
![이미지_205](https://github.com/exult/exult/assets/152712037/3b944931-d236-4873-a71e-23e07177b5ac)
![이미지_279](https://github.com/exult/exult/assets/152712037/2d2e5e8d-c3f5-4f97-be00-99fc4d498976)

If you want enjoy my things, You can get it from my blog, 

Here is the link. 

https://blog.naver.com/sohnmkie/222911698649

If This is second mention, I'm sorry ^^;;;;;

**Comment by DominusExult** [comment]
heya, cool stuff! thanks for sharing!

**Reply by SMK-SonHR** [reply]
Thank you :)

**Comment by DominusExult** [comment]
the mouse pointers are also interesting!

**Comment by KnightCaptainU7** [comment]
I'd love to add these to my mod. Can you make a dead body shape for each humanoid?

**Comment by SMK-SonHR** [comment]
ok ^^

I will make some blood pixel on each character's sleep image :)

Thanks for your interest :)

Thank you ^^

-----Original Message-----

**Comment by SMK-SonHR** [comment]
just put some red dots on image13
Thax :)


-----Original Message-----

**Reply by KnightCaptainU7** [reply]
Did you post them on your page? Or just updated the zip files?

Do you have the "in-hand offsets" of X, Y for the weapons? Or can you tell me which shapes you based each one on?

**Reply by KnightCaptainU7** [reply]
Also, thanks for doing that!

**Comment by thecommissar** [comment]
These are so awesome. Can I use the U8-style Avatar in my mod (maybe with a slight modification)?

**Comment by SMK-SonHR** [comment]
YES! ^^;;;
I owe everything here. 
My hobby is just play u7 on exult and make things like them
Everyone can use my things. 
My honor ^^

-----Original Message-----

**Reply by thecommissar** [reply]
Thanks so much!

Do you have Discord? Is there a way I can contact you privately? I have something I'd like to run by you.

**Comment by SMK-SonHR** [comment]
oh, sorry. i'm late. 

um... honestly, I' don't know about each offset 
But I think you can fix it. I'm sorry

Thanks. :)


-----Original Message-----

**Comment by SMK-SonHR** [comment]
I dont have discord :)
my e-mail is ***@***.***"
Thax :) 
-----Original Message-----

**Reply by thecommissar** [reply]
Hey! Github censors your email address, so I DM'ed you I think in nexusmods.com!

**Comment by SMK-SonHR** [comment]
No, I don't have account Nexusmods.com 
sorry ^^;
But I can reply when you ask something

-----Original Message-----

**Reply by thecommissar** [reply]
I was wondering if you'd be interested in helping make more U7 shapes for a mod project. But I think it would be easier to communicate about if we can figure out how to contact each other directly.
